aws.yaml

```diff
@@ -1,3 +1,49 @@
+- active: false
+  apps:
+    - app: chart-operator
+      version: 0.11.4
+    - app: coredns
+      version: 1.1.3
+      componentVersion: 1.6.5
+    - app: cert-exporter
+      version: 1.2.1
+    - app: kube-state-metrics
+      version: 1.0.0
+      componentVersion: 1.9.0
+    - app: metrics-server
+      version: 1.0.0
+      componentVersion: 0.3.3
+    - app: net-exporter
+      version: 1.5.1
+    - app: node-exporter
+      version: 1.2.0
+      componentVersion: 0.18.1
+    - app: cert-manager
+      version: 1.0.4
+      componentVersion: 0.9.0
+    - app: cluster-autoscaler
+      version: 1.1.4
+      componentVersion: 1.16.2
+    - app: external-dns
+      version: 1.2.0
+      componentVersion: 0.5.18
+    - app: kiam
+      version: 1.1.0
+      componentVersion: 3.5.0
+  authorities:
+    - name: app-operator
+      version: 1.0.0
+    - name: aws-operator
+      version: 8.1.0
```

Comment:
This should also be WIP. https://github.com/giantswarm/aws-operator/blob/de013a3681525e3b47525a7e57989955c87ad226/pkg/project/project.go#L4